python/uagents-core/uagents_core/utils/registration.py

```diff
@@ -77,6 +121,8 @@ def register_in_almanac(
 
     Args:
         identity (Identity): The identity of the agent.
+        prefix (str | None): The Almanac contract where the agent will be registered.
+            If none is provided, the agent will be registered to the default contract.
```

Comment:
Suggest simplifying this since registering to Alamanc API does not actually require any contract interactions.
```suggestion
        prefix (Literal["agent", "test-agent"] | None): The prefix for the agent identifier.
```